documents/TestSuite/Utilities/constant_color_scene.xml

```diff
@@ -1,6 +1,6 @@
 <!-- Template for taking an input color shader and rendering without lighting -->
 <World>
-   <Camera eye="0, 2.3, 2.3" look_at="0,0,0" fov="70" />
+   <Camera eye="11.5, 0, 0" look_at="0,0,0" fov="10" />
```

Comment:
As mentioned, it's kinda strange that I needed to set this to such a low number to get things to look similar. A bit confused by it.